### DISCUSSION TITLE
Feature Request - AI Summaries

### DISCUSSION THREAD

**Discussion by Hottorun** [karakeep-app/karakeep#367] [Ideas]
As the title says, I thought it might be a good idea to be able to click on like the picture to go to the website/URL. Clicking the name would display a hovering view of the website on one side and then the title, URL, tags, and an AI summary. Sort of like mymind does it (look at the screenshot for example) 
![image](https://github.com/user-attachments/assets/69630ada-1266-42e4-a77a-c5572e1fe0a7)

**Comment by MohamedBassem** [comment]
AI summaries should be pretty easy to do. Though I didn't want them to be enabled by default because they'll increase the OpenAI cost. I guess we can make them configurable, and if you don't have them enabled for every URL by default, you can request them on demand. I like that.
